src/utils/sendEvent.js

```diff
@@ -1,22 +1,12 @@
-const BASE_URL = 'https://en.wikipedia.org/beacon/event'
-const MAX_URL_LENGTH = 2000
+import { isProd } from 'utils'
 
-const isUrlValid = url => url.length <= MAX_URL_LENGTH
+const intakeUrl = 'https://intake-analytics.wikimedia.org/v1/events' + (isProd() ? '?hasty=true' : '')
 
-const buildBeaconUrl = event => {
-  const queryString = encodeURIComponent(JSON.stringify(event))
-  return `${BASE_URL}?${queryString}`
-}
-
-export const sendEvent = (schema, revision, language, event) => {
-  const url = buildBeaconUrl({
-    schema,
-    revision,
-    event,
-    webHost: `${language}.wikipedia.org`,
-    wiki: `${language}wiki`
-  })
-  if (isUrlValid(url)) {
-    navigator.sendBeacon(url)
-  }
+export const sendEvent = ($schema, stream, event) => {
+  const body = JSON.stringify([{
+    $schema,
```

Comment:
I gather the `$` is just a naming convention for the schema?

https://wikitech.wikimedia.org/wiki/Event_Platform/EventLogging_legacy#produce_event_data